app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -158,6 +159,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.ssh_keypair_userid              = data.ssh_keypair_userid;
 
       $scope.emsCommonModel.service_account                 = data.service_account;
+      $scope.emsCommonModel.default_service_account         = data.service_account;
```

Comment:
and here (https://github.com/ManageIQ/manageiq-ui-classic/pull/5621/files#r322236441)

`data.service_account` should not mean 2 different things

Reply:
See the line above - it's for GCE. I was trying to stay with it, but you asked me to use `default_` prefix - here you are.

Reply:
That prefix is needed for any value in the Default authentication tab. It has nothing to do with how the value is named in the database.